### ISSUE TITLE
Window position not saved correctly when restart

### ISSUE THREAD

**Issue by raelb** [qutebrowser/qutebrowser#4766]
**Version info (see `:version`)**: v1.6.2 x64 Standalone version

**Description**

I'm using qutebrowser on Windows 10.17763. After I close program and restart, the Window position is about 1 cm lower than before. Each time I restart the Window goes 1 cm lower.

Using 150% scaling.
Resolution 1920x1080

**Comment by The-Compiler** [comment]
@raelb If you still use qutebrowser, is this something you still see with an up-to-date Qt/qutebrowser version?

**Comment by raelb** [comment]
Yes, the problem still exists with qutebrowser-1.12.0-windows-standalone-amd64.

**Comment by The-Compiler** [comment]
Thanks for the update! I suspect it's qutebrowser's window geometry saving/loading causing this somehow together with HighDPI, but no idea what's going wrong exactly.

**Comment by openandclose** [comment]
Hello, I'm using arch linux and I've been having the same problem for a year or two (or more, sorry, I don't remember).

And I was recently surprised
when I run windows 10 on a different PC, installed qutebrowser, and I had the same problem.

So I don't think it is related to OS,
maybe to some user configuration (config.py).

To be exact, my case is about 7-8 mm lower,
about the length of title bar (plus the window pane).
(I'm using just ``openbox`` window manager, no desktop environment).

---

Qutebrowser version: v1.13.1
Display Resolution 1920x1080

c.auto_save.session = True
c.zoom.default = '125%'
c.tabs.position = 'left'

**Comment by openandclose** [comment]
I forgot to do a basic test.

It was the same when I renamed ``config.py`` temporaliry and run qutebrowser
(with default config, tabs on top).

**Comment by trysdyn** [comment]
This is still an issue in 2.2.0 on Windows 10. A trifling one but slightly irritating for my use case :)

I did some testing and found that qute moves down exactly the height of a titlebar each re-run, even on a clean install from release binary and no user configs. I believe something is not being consistent in whether it's dealing with decorated or undecorated window geometry but a quick peek didn't turn up what. It just looks like all Qute does is wrap QWidget for all of its geometry functions.
